<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Talking</class>
 <widget class="QWidget" name="Talking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="list_friend">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>141</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_talkScreen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_inputScreen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_B">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_I">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
